sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -36,10 +36,31 @@
     private List<Permission> permissions;
     private TokenResolver tokenResolver;
     private CosmosKeyCredential cosmosKeyCredential;
+    private boolean sessionCapturingOverride;
 
     public CosmosClientBuilder() {
     }
 
+    /**
+     * Session capturing is enabled by default for {@link ConsistencyLevel#SESSION}.
+     * For other consistency levels, it is not needed, unless if you need occasionally send requests with Session
+     * Consistency while the client is not configured in session.
+     *
+     * enabling Session capturing for Session mode has no effect.
+     * @param sessionCapturingOverride the session capturing override
```

Comment:
We should mention default value of this parameter here. 
It is `false` by default, and that is the intention, right ?

Reply:
sure, will mention here. yes by default it is false

Reply:
correct. thanks for pointing that out. will update javadoc shortly.

Reply:
done.